### ISSUE TITLE
Bug: Positional parameters are not supported.

### ISSUE THREAD

**Issue by hden** [metabase/metabase#6959]
`?` marks in a custom query is somehow interpreted as positional parameters.

Minimal required SQL
```sql
#standardSQL
-- ? Removing this comment will resolve the error
select id
from tupac_sightings.sightings
where timestamp_seconds(timestamp) between {{start_date}} and {{end_date}}
```

Error message: `Positional parameters are not supported at [6:86]`

-------
- Your browser and the version: Chrome 63.0
- Your operating system: OS X 10
- Your databases: BigQuery
- Metabase version: v0.28.0
- Metabase hosting environment: jar
- Metabase internal database: MySQL
-------

**Comment by flamber** [comment]
This sounds related to #7742 and would be fixed by #8674 (if it gets merged)

**Comment by flamber** [comment]
I cannot reproduce on 0.35.4 - likely fixed by the change to parameterized queries for BigQuery
